plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3ConfigurationInitializer.java

```diff
@@ -211,6 +215,9 @@ public void initializeConfiguration(Configuration config)
         config.set(S3_MAX_BACKOFF_TIME, maxBackoffTime.toString());
         config.set(S3_MAX_RETRY_TIME, maxRetryTime.toString());
         config.set(S3_CONNECT_TIMEOUT, connectTimeout.toString());
+        if (connectTtl.isPresent()) {
```

Comment:
```suggestion
connectTtl.ifPresent(duration -> config.set(S3_CONNECT_TTL, duration.toString()));
```

Reply:
> 

I have modified the code at this location, thank you~ Intellij IDEA suggested to change it to this lambda expression way, but I thought it might should be consistent with the format of other if clauses in `TrinoS3ConfigurationInitializer` , so I didn’t change it the first time.